.github/workflows/pytest-release-with-uv.yml

```diff
@@ -0,0 +1,88 @@
+name: Tests
+
+on:
+  workflow_call:
+    inputs:
+      python-version:
+        description: Python version
+        type: string
+        default: "3.9"
+      os:
+        description: Operating system
+        type: string
+        default: ubuntu-latest
+      pre-install-cmd:
+        description: Run before install
+        type: string
+        default: ""
+      install-uv:
+        description: Install uv
+        type: string
+        default: "pip install uv"
+      install-cmd-uv:
+        description: Install dependencies with uv
+        type: string
+        default: "uv pip install -e '.[test]'  --system"
```

Comment:
This is what I added. All others are mostly just copied.

Reply:
reusable GH workflows are a very nice feature. essentially means all my personal GH repos can be updated to use `uv` in CI by changing this 1 line 😄